### PR TITLE
bold, italic issue resolved

### LINKED ISSUE #8589: Text turns bold automatically when italics key is turned off in text editor component
Found a bug? Please fill out the sections below. 👍

### Issue Summary

The rich text editor component changes the text to bold automatically when the italics key is turned off.
Preview of the bug:
          

https://user-images.githubusercontent.com/75200954/235377213-6d4625e5-4734-4d93-90be-92d222f0d5f5.mp4

### Steps to Reproduce

1. Went to https://app.cal.com/event-types?dialog=new&eventPage=zoheb-ahmed-zlwmgk
2. Clicked on bold and italics alternatively

Any other relevant information. For example, why do you consider this a bug and what did you expect to happen instead?

Expected behaviour:
Preview of the expected behaviour:-

https://user-images.githubusercontent.com/75200954/235377216-43e34cbd-8a9a-4213-b0ce-9e9992797d27.mp4

### Technical details

- Browser version, screen recording, console logs, network requests: You can make a recording with [Bird Eats Bug](https://birdeatsbug.com/).
- Node.js version
- Anything else that you think could be an issue.

### PR DESCRIPTION
**Pull request by Inderjit-fullstack-dev**
## What does this PR do?

While adding a new Even, When the user toggle between Bold and Italic style, it was not working properly

Fixes #8589

https://www.loom.com/share/49a60df92a4e412e96227b5e7129f7a9

**Environment**: Staging(main branch)

## Type of change

- Bug fix (non-breaking change which fixes an issue)

## How should this be tested?
This can be tested using following steps
1. Login to portal
2. Click on Events
3. Add new Event
4. Toggle between Bold and Italic

### CODE REVIEW

**Review by Udit-takkar — DISMISSED** [review]
LGTM. Thanks for your contribution

File: packages/ui/components/editor/plugins/ToolbarPlugin.tsx [hunk 1]
```diff
@@ -474,7 +474,7 @@ export default function ToolbarPlugin(props: TextEditorProps) {
               StartIcon={Italic}
               onClick={() => {
                 editor.dispatchCommand(FORMAT_TEXT_COMMAND, "italic");
-                if (isItalic) {
+                if (isBold) {
```
Comment (Udit-takkar) [inline packages/ui/components/editor/plugins/ToolbarPlugin.tsx:477]: @Inderjit-fullstack-dev maybe we can remove these if conditions from this onClick function and https://github.com/calcom/cal.com/blob/6e55db3970efe44b3606cfb9658031b05f30ea41/packages/ui/components/editor/plugins/ToolbarPlugin.tsx#L462

Comment (hariombalhara) [reply packages/ui/components/editor/plugins/ToolbarPlugin.tsx:477]: Agreed with @Udit-takkar 
I don't see any issue with removing the bold check from italic and italic check from bold. Though there has to be some reason for adding that. 

Blocking it till @CarinaWolli confirms.

**Review by hariombalhara — CHANGES_REQUESTED** [review]
Left a comment

**Comment by Udit-takkar** [comment]
closing this because this PR has been merged now https://github.com/calcom/cal.com/pull/8613

**Pull request closed without merging**